DBM-Core/modules/objects/Timer.lua

```diff
@@ -749,18 +756,18 @@ function timerPrototype:Update(elapsed, totalTime, ...)
 			local countVoice = self.mod.Options[self.option .. "CVoice"] or 0
 			if (type(countVoice) == "string" or countVoice > 0) then
 				if not bar.fade then--Don't start countdown voice if it's faded bar
-					if newRemaining > 2 then
+					if newMinRemaining > 2 then
 						--Can't be called early beacuse then it won't unschedule countdown triggered by :Start if it was called
 						--Also doesn't need to be called early like it does in AddTime and RemoveTime since those early return
 						DBM:Unschedule(playCountSound, id)
-						playCountdown(id, newRemaining, countVoice, self.countdownMax, self.requiresCombat)--timerId, timer, voice, count
+						playCountdown(id, newMinRemaining, countVoice, self.countdownMax, self.requiresCombat)--timerId, timer, voice, count
 						DBM:Debug("Updating a countdown after a timer Update call for timer ID:" .. id)
 					end
 				end
 			end
 		end
 		local updated = DBT:UpdateBar(id, elapsed, totalTime)
-		test:Trace(self.mod, "UpdateTimer", self, id, elapsed, totalTime)
+		test:Trace(self.mod, "UpdateTimer", self, id, elapsed, (correctedTimer or totalTime)) -- REVIEW!
```

Comment:
@MysticalOS , this is where I'd need some help, since I don't know what Trace/Testing framework does nor how it behaves.
Does it expect a number always, or could it be a string?